src/promptflow-evals/promptflow/evals/evaluators/chat/__init__.py

```diff
@@ -4,249 +4,8 @@
 
 __path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore

```

Comment:
Can we use this chance to move this line?

Reply:
Where should we move this ?

Reply:
Sorry. I mean remove this line. Type too fast ;)